frontend/common/src/components/Alert/Alert.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from "react";
+
+export type Color = "success";
+
+export interface AlertProps {
+  title: string;
+  message: string;
+  icon: React.ReactNode;
+  type: "success";
+}
```

Comment:
for ease of intl-ing, maybe instead of strings have them as ReactNodes so you can pass in intls as props?

Reply:
this threw no errors by the way, but I guess for styling purposes and consistency you might want to add the type flexibility as we've done in other components
![image](https://user-images.githubusercontent.com/40485260/172704946-8bd4ceb2-9cb6-4b79-82b5-c61c72d79cf6.png)

Reply:
> pre-fill doesn't seem to happen, not even for preferred language -> how much prefill should happen?

The English in preferred language should be selected by default. It's working on my end 😕 
![image](https://user-images.githubusercontent.com/22059495/172908395-86e80629-c434-4fc7-b1b4-bc1ca8ffc299.png)